frontend/src/pages/rules/[rule]/edit-rule.tsx

```
@@ -0,0 +1,113 @@
+    if (rule.name) {
+      setIsLoading(true);
+      try {
+        const savedRuleName = await rulesStore.save(rule);
+        const isWithErrors = !!rule.error?.message;
+        notificationsStore.showNotification({
+          variant: isWithErrors ? 'warn' : 'success',
+          text: isWithErrors ? t('rules.labels.success-errors') : t('rules.labels.success'),
+        });
+
+        if (pathName === 'new') {
+          return location.replace('/#!/rules/edit/' + savedRuleName);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)